config/crd/bases/rabbitmq.com_rabbitmqclusters.yaml

```diff
@@ -1,13 +1,3 @@
-# RabbitMQ Cluster Operator
-#
-# Copyright 2020 VMware, Inc. All Rights Reserved.
-#
-# This product is licensed to you under the Mozilla Public license, Version 2.0 (the "License").  You may not use this product except in compliance with the Mozilla Public License.
-#
-# This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
-
-
----
```

Comment:
Looks like an inadvertent change. Let's roll it back 🗞️

Reply:
Added the notice.
It got removed because `make manifests` removed it. Fixed in #305.